### ISSUE TITLE
empty objects are persisted to the source code randomly

### ISSUE THREAD

**Issue by lhein** [KaotoIO/kaoto-next#388]
See exceptionHandler and filehasher parameters appearing in below video out of nowhere...

![Image](https://github.com/KaotoIO/kaoto-next/assets/435853/bfb3d540-3cbf-4b47-aced-0523816c5d77)

**Comment by lordrip** [comment]
Related route:

```
- route:
    from:
      uri: file-watch
      steps:
        - log:
            message: template message
      id: from-2648
    id: route-3021
```

This seems related to the default value for `objects` properties from `uniforms-patternfly`

**Comment by lhein** [comment]
fixed by #511
